src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -450,7 +449,7 @@ private void EnsureBackingFieldIsSynthesized()
             }
 
             // If binding both the getter and setter didn't get a backing field, set it to null so that we don't re-calculate.
-            Interlocked.CompareExchange(ref _lazyBackingFieldSymbol, null, _lazyBackingFieldSymbolSentinel);
+            MarkBackingFieldAsCalculated();
```

Comment:
>MarkBackingFieldAsCalculated [](http://example.com/codeflow?start=12&length=28)

How is this change related to the goal of this PR. Please keep things separate. If you would like to make this refactoring, please submit a separate PR.

Reply:
@AlekseyTs This was addressing @333fred feedback that was asked to be done in the next PR. I'm okay with addressing the feedback separately.

Reply:
> This was addressing feedback that was asked to be done in the next PR. I'm okay with addressing the feedback separately.

The definition of the next PR is quite vague. Is it the next submitted PR, or is it the next merged PR? These are different things. So, in general, we understand that the term "next PR" refers to a PR in a **near future** that addresses the concern.